Source/PrivateHeaders/Ably/ARTTestClientOptions.h

```diff
@@ -31,6 +31,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic) id<ARTRealtimeTransportFactory> transportFactory;
 
+/**
+ Initial value is `nil`.
```

Comment:
Can you please document what this property does and what it's for? It sounds like it's used to provide a way for test code to be able to simulate the case where a reconnection attempt results in a different outcome to the original connection attempt, is that right?

Reply:
https://github.com/ably/ably-cocoa/pull/1703/commits/08ba4c8fded2d94ec7efcecba95ccf979b9c80ca